### PR TITLE
More stat unlocks

### PR DESCRIPTION
**Pull request by LordByronRS**
Added a few missing stats and modified some values.

### CODE REVIEW

File: WasabiWords.lua [hunk 1]
```diff
@@ -298,7 +322,12 @@ wasabi_words:add_button("ShinyWasabi", function() --Original script by ShinyWasa
         stats.set_int('MPX_CHAR_HAIR_UNLCK9', -1)
         stats.set_int('MPX_CHAR_HAIR_UNLCK10', -1)
         stats.set_int('MPX_CHAR_HAIR_UNLCK11', -1)
-        stats.set_int('MPX_CHAR_HAIR_UNLCK1', -1)
```
Comment (gir489returns) [inline WasabiWords.lua:301]: Why was this removed?

**Review by gir489returns — CHANGES_REQUESTED** [review]

File: WasabiWords.lua (hunk 1 above)
Comment (LordByronRS) [reply WasabiWords.lua:301]: Just because it was a duplicate. It was probably meant to be `MPX_CHAR_HAIR_UNLCK12`, but the 2 was missing. The same stat is still at the beginning of that block.

Comment (gir489returns) [reply WasabiWords.lua:301]: Yeah I saw when I checked the file. Thanks for the contribution.
